src/roulette.py

```diff
@@ -126,3 +126,172 @@ def get(self, bin: int) -> Bin:
         :rtype: Bin
         """
         return self.bins[bin]
+
+
+class BinBuilder:
+    def __init__(self) -> None:
+        pass
+
+    def buildBins(self, wheel: Wheel) -> None:
+        self.create_straight_bets(wheel)
+        self.create_horizontal_split_bets(wheel)
+        self.create_vertical_split_bets(wheel)
+        self.create_street_bets(wheel)
+        self.create_corner_bets(wheel)
+        self.create_line_bets(wheel)
+        self.generate_five_bets(wheel)
+        self.create_even_money_bets(wheel)
+        self.create_dozen_bets(wheel)
+        self.create_column_bets(wheel)
+
+    @staticmethod
+    def create_straight_bets(wheel: Wheel) -> None:
+        straight_bet_odds = 35
+        straight_bet_numbers = {_ for _ in range(37)}
+
+        for number in straight_bet_numbers:
+            straight_bet = Outcome(str(number), straight_bet_odds)
```

Comment:
`straight_bet` is an `Outcome` object, not some bet class object. Better rename to `straight_bet_outcome` or simply `outcome`.

Reply:
Done. Thanks.